File: Domain/SuggestionActivity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Domain
{
    public class SuggestionActivity
    {
        public Guid Id { get; set; }//gjenerimi i id nga db

        public string Title { get; set; }

        public DateTime Date { get; set; }

        public String Category { get; set; }

        public string Description { get; set; }

        public string City { get; set; }

        public string Venue { get; set; }


    }
}

File: client-app/src/app/models/suggestionactivity.ts
import { Profile } from "./profile";
export interface SuggestionActivity {
    id: string;
    title: string;
    date: Date | null;
    description: string;
    category: string;
    city: string;
    venue: string;
    host?: Profile;
    hostUsername: string;
    isHost: boolean;
    attendees: Profile[]
  }


  export class SuggestionActivity implements SuggestionActivity {
    constructor(init?: SuggestionActivityFormValues){
      Object.assign(this,init)
    }
  }

  export class SuggestionActivityFormValues {
    id?: string = undefined;
    title: string= '';
    category: string= '';
    description: string= '';
    date: Date | null = null;
    city: string = '';
    venue: string = '';

    constructor(activity?: SuggestionActivityFormValues){
      if(activity) {
        this.id = activity.id;
        this.title = activity.title;
        this.category = activity.category;
        this.description = activity.description;
        this.date = activity.date;
        this.venue = activity.venue;
        this.city = activity.city;
      }
    }
  }
  